pkg/models/registries/image_type.go

```diff
@@ -0,0 +1,94 @@
+package registries
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"time"
+)
+
+type AuthInfo struct {
+	Username   string `json:"username" description:"username"`
+	Password   string `json:"password" description:"password"`
+	ServerHost string `json:"serverhost" description:"registry server host"`
+}
+
+type ImageNameAndSecret struct {
+	ImageName string        `json:"imageName,omitempty" description:"Image full name."`
+	Secret    corev1.Secret `json:"secret,omitempty" description:"Secret holds secret data of a certain type."`
+}
+
+// ImageBlobInfo describes the info of an image.
+type ImageBlobInfo struct {
+	// Status is the status of the image search, such as "failed","succeeded".
+	Status string `json:"status" description:"Status is the status of the image search, such as \"failed\",\"succeeded\"."`
+	// Image is the metadata of that image
+	Image *ImageBlob `json:"imageBlob,omitempty" description:"Retrieve the blob from the registry identified. Reference: https://docs.docker.com/registry/spec/api/#blob"`
+}
+
+type ImageBlob struct {
+	Architecture    string          `json:"architecture,omitempty" description:"The architecture field specifies the CPU architecture, for example amd64 or ppc64le."`
+	Config          Config          `json:"config,omitempty" description:"The config field references a configuration object for a container."`
+	Container       string          `json:"container,omitempty" description:"Container id."`
+	ContainerConfig ContainerConfig `json:"container_config,omitempty" description:"The config data of container."`
+	Created         time.Time       `json:"created,omitempty" description:"Create time."`
+	DockerVersion   string          `json:"docker_version,omitempty" description:"docker version."`
```

Comment:
json field name use camel case style, and starts with lower case letter

Reply:
this json field name can not modify, here are docker registry response data. https://docs.docker.com/registry/spec/api/#blob